nav2_controller/include/nav2_controller/nav2_controller.hpp

```diff
@@ -229,7 +229,7 @@ class ControllerServer : public nav2_util::LifecycleNode
   double min_y_velocity_threshold_;
   double min_theta_velocity_threshold_;
 
-  double controller_patience_;
+  double failure_tolerance_;
```

Comment:
Please add this to the nav2_params.yaml file with some reasonable default (idk, 0.50 seconds?)

Also, add the new param to the configuration guide for the controller server (including what `-1` means)

Reply:
Also, the migration guide that this feature is now available

Reply:
> Please add this to the nav2_params.yaml file with some reasonable default (idk, 0.50 seconds?)

Just added 0.3, but I could also set it to 0.0 if you want to have the exact same behavior as before this PR (0.0 disables it), what do you think?

Reply:
0.3 works